microsoft/testsuites/core/storage.py

```diff
@@ -374,6 +376,63 @@ def verify_azure_file_share_nfs(self, log: Logger, node: Node) -> None:
         finally:
             nfs.delete_share()
             node.tools[NFSClient].stop(mount_dir)
+    
+    _file_path = "~/SAMPLE.txt"
```

Comment:
These should be `CAPITALIZED_SNAKE_CASE`

Reply:
Should be addressed now